packages/orbit-components/src/Textarea/index.js

```diff
@@ -128,7 +128,7 @@ const Textarea = React.forwardRef<Props, HTMLElement>(
       name,
       rows,
       readOnly,
-      tabIndex,
+      tabIndex = 0,
```

Comment:
I'm not against this, but it's the same as `undefined`, no? Because `textarea` is focusable by default.